### PR TITLE
[Typescript jQuery] Add README.md

### PR DESCRIPTION
**Pull request by JFCote**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

@TiFu (2017/07) @taxpon (2017/07) @sebastianhaas (2017/07) @kenisteward (2017/07) @Vrolijkx (2017/09)

### Description of the PR

Add the readme.md that I think was removed when I've done the refactoring a couple of months ago.

### CODE REVIEW

File: modules/swagger-codegen/src/main/resources/typescript-jquery/README.mustache [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+To build an compile the typescript sources to javascript use:
+```
+npm install
+npm run build
+```
+
+### Publishing
+
+First build the package than run ```npm publish```
```
Comment (TiFu) [inline modules/swagger-codegen/src/main/resources/typescript-jquery/README.mustache]: I think `than`  should be  `then`

File: modules/swagger-codegen/src/main/resources/typescript-jquery/README.mustache [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Consuming
+
+navigate to the folder of your consuming project and run one of next commando's.
```
Comment (TiFu) [inline modules/swagger-codegen/src/main/resources/typescript-jquery/README.mustache]: `commands`? (without apostrophe and o)

File: modules/swagger-codegen/src/main/resources/typescript-jquery/README.mustache (hunk 1 above)
Comment (JFCote) [reply modules/swagger-codegen/src/main/resources/typescript-jquery/README.mustache]: You are right, I did a copy paste from another readme.mustache so I didn't check for error like this. Will fix in my next commit. Thanks

File: modules/swagger-codegen/src/main/resources/typescript-jquery/README.mustache (hunk 2 above)
Comment (JFCote) [reply modules/swagger-codegen/src/main/resources/typescript-jquery/README.mustache]: will fix in my next commit

**Comment by JFCote** [comment]
@TiFu The fix is done!

**Pull request merged**